From 02a4fcc2c46dc51ccb1ba3c186cc453950adc1f5 Mon Sep 17 00:00:00 2001
From: Gioele Barabucci <gioele@svario.it>
Date: Sun, 27 Nov 2022 10:58:35 +0100
Subject: [PATCH] d/rules: Use SOURCE_DATE_EPOCH instead of dpkg-parsechangelog

Using the debhelper-provided `SOURCE_DATE_EPOCH` allows avoiding
one call to `dpkg-parsechangelog` and fixes the lintian warning:

    I: debian-keyring source: debian-rules-parses-dpkg-parsechangelog
---
 debian/rules | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/debian/rules b/debian/rules
index c273ca4a..29abbe6d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,11 +1,9 @@
 #!/usr/bin/make -f
 
-BUILD_DATE := $(shell dpkg-parsechangelog -SDate)
-
 %:
 	dh $@
 
 override_dh_clean:
 	dh_clean
 	mkdir -p output/keyrings
-	touch --date='$(BUILD_DATE)' output output/keyrings debian
+	touch --date='@$(SOURCE_DATE_EPOCH)' output output/keyrings debian
-- 
2.35.1

